packages/transformers/postcss/src/loadConfig.js

```diff
@@ -0,0 +1,149 @@
+// @flow
+import type {Config, PluginOptions} from '@parcel/types';
+import type {PluginLogger} from '@parcel/logger';
+import path from 'path';
+
+import loadExternalPlugins from './loadPlugins';
+
+const MODULE_BY_NAME_RE = /\.module\./;
+
+async function configHydrator(
+  configFile: any,
+  config: Config,
+  options: PluginOptions,
+) {
+  // Use a basic, modules-only PostCSS config if the file opts in by a name
+  // like foo.module.css
+  if (configFile == null && config.searchPath.match(MODULE_BY_NAME_RE)) {
+    configFile = {
+      plugins: {
+        'postcss-modules': {},
+      },
+    };
+  }
+
+  if (configFile == null) {
+    return;
+  }
+
+  // Load the custom config...
+  let modules;
+  let configPlugins = configFile.plugins;
+  if (
+    configPlugins != null &&
+    typeof configPlugins === 'object' &&
+    configPlugins['postcss-modules'] != null
+  ) {
+    modules = configPlugins['postcss-modules'];
+    // $FlowFixMe
+    delete configPlugins['postcss-modules'];
+  }
+
+  if (!modules && configFile.modules) {
+    modules = {};
+  }
+
+  let plugins = await loadExternalPlugins(
+    configPlugins,
+    config.searchPath,
+    options,
+  );
+
+  return config.setResult({
+    raw: configFile,
+    hydrated: {
+      plugins,
+      from: config.searchPath,
+      to: config.searchPath,
+      modules,
+    },
+  });
+}
+
+export async function load({
+  config,
+  options,
+  logger,
+}: {|
+  config: Config,
+  options: PluginOptions,
+  logger: PluginLogger,
+|}) {
+  let configFile: any = await config.getConfig(
+    ['.postcssrc', '.postcssrc.json', 'postcss.config.js', '.postcssrc.js'],
+    {packageKey: 'postcss'},
```

Comment:
`package.json` key config seems to be broken from the test I wrote, not entirely sure if this is a known issue or if I just use it incorrectly

Reply:
I don't think I noticed that. I just copied `getConfig` from the Asset class, maybe something broke along the way.